images/str-r/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-base:1.1
```

This should start from the `r` driver base:

```suggestion
FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-r:1.1
```
```suggestion
FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-base:1.1
```